### ISSUE TITLE
Was just curious if there was any ETA on release 022? :)

### ISSUE THREAD

**Issue by AerowynX** [KozGit/DOOM-3-BFG-VR#158]
just wondering wanted to write out a guide on some things but was waiting for the next release

**Comment by KozGit** [comment]
Hey, I just committed the new player hand models and anims - doing some testing now.
I'm not an artist/animator, so it took way longer than I expected, sorry!

I just want to get it tested and the push the release - stay tuned!

**Comment by AerowynX** [comment]
awesome was just curious ill hold off on my guide till it's done:)

**Comment by AerowynX** [comment]
still waiting ha:P

**Comment by AerowynX** [comment]
doing alright man? do i need to alert the authorities make sure you are alive and well????? :)

**Comment by KozGit** [comment]
Man, I'm really sorry.  No need to call the authorities, but thanks for checking! :) 
I've been working on the playermodel and IK, and to be honest the IK was hurting my head a little.  
I'm knocking off the rough edges to the changes now.  I'm fixing one more thing before release, and thats the player body direction when moving using room scale.  

I know it doesn't look like a lots being done, but the player model and anim updates along with figuring out the IK a little better was a **ton** of work - I've been putting in time on it every day.
If I'm lucky I'll get a test build out by Monday.

Hey, also looking for some feedback - the current organization of the vr options menus was designed around a limit of only being able to have 8 options per menu screen.  I modified the code to allow scrolling in the menus to avoid this limitation.  It seems like there is probably a better way to group all the options, but I don't wan't to confuse people who are used to it the way it is.   ( Also it would make any existing guides inaccurate.) What are your thoughts on re-grouping menu options?

**Comment by AerowynX** [comment]
no worries man! honestly not sure why you even bother putting time into a full player model i bet most people just play with hands only anyway:P 

in terms of menu options i don't really have much of a preference as long as options are selectable in VR ingame.. the worst part of the other mod was all the options were command based was big pain in the ass

**Comment by KozGit** [comment]
Most of the player model updates were actually for the player hands ( shown even when no body  ) - updated the hands to start supporting finger poses.

I must be one of the weirdos who likes to have a full body :)

**Comment by AerowynX** [comment]
oh nice yea hands is the most important:) I don't like full body in any game I have tried that has it.. without full trackable limbs it just feels so off it breaks immersion more than not having a visible body at all

**Comment by SimplexPL** [comment]
You probably hate that question, but any ETA on next version? :)

**Comment by AerowynX** [comment]
hopefully these are the best looking hands ever! lol :P

**Comment by Nordomus** [comment]
So how is it going? I'm actually stuck in Codes4fun mod in Hell because it is broken so I was hoping to switch to this one but I need same great locomotion system and black borders in cutscenes, those are must for me. Also are the saves compatible?

**Comment by AerowynX** [comment]
this one has way more locomotion options

**Comment by Nordomus** [comment]
Yeah but I read in some other topic that speed curve is not that good with trackpad locomotion. I guess I gonna have to test it myself. What about save files?

**Comment by AerowynX** [comment]
not sure i did a clean install for this one

**Comment by AerowynX** [comment]
i guess i should give these keys away ha.. hopefully they still work

**Comment by KozGit** [comment]
Hold tight! Im not dead! :)

**Comment by KozGit** [comment]
> What about save files?

Save files are not compatible between the different mods.  And if you install a hi def pack on either mod, the save games from before and after the hi def install will not be compatible either.  This is a limitation of the way BFG saves games - they're kind of a big binary dump, and changes made to the game after a save, especially to any of the scripts, render them incompatible with each other.

**Comment by AerowynX** [comment]
haha i know man i been following along i check this github pretty much everyday :P
